src/star/star_genomegenerate/config.vsh.yaml

```
@@ -0,0 +1,138 @@
+name: star_genomegenerate
```

I wonder if we should call this just `generate_genome` for instance and add the namespace `star` instead?

TBD

---

well, `name: star_genomegenerate` is in line with other components in biobase.

I might consider 

```suggestion
name: star_genome_generate

---

If we stick to `start_...` as a conventions, that's fine by me.

In line with `star_align_reads` I would propose `star_generate_genome`.

---

Renamed the component.